{"uid":"815570a836cc98b6","name":"test 04.02.07 create an account with incorrect confirm password","fullName":"tests.test_create_an_account.TestCreateAnAccount#test_tc_04_02_07_create_account_with_incorrect_confirm_password","historyId":"09679f850e6cf760d69d28578925e32f","time":{"start":1697332619317,"stop":1697332624441,"duration":5124},"description":" Verify that customer can't Create An Account with incorrect confirm password","descriptionHtml":"<p>Verify that customer can't Create An Account with incorrect confirm password</p>\n","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x557f3a6946b3 <unknown>\n#1 0x557f3a36a1e7 <unknown>\n#2 0x557f3a3b1566 <unknown>\n#3 0x557f3a3b1651 <unknown>\n#4 0x557f3a3eea74 <unknown>\n#5 0x557f3a3d30ed <unknown>\n#6 0x557f3a3ec322 <unknown>\n#7 0x557f3a3d2e93 <unknown>\n#8 0x557f3a3a5934 <unknown>\n#9 0x557f3a3a671e <unknown>\n#10 0x557f3a659cb8 <unknown>\n#11 0x557f3a65dbf0 <unknown>\n#12 0x557f3a66819c <unknown>\n#13 0x557f3a65e808 <unknown>\n#14 0x557f3a62b27f <unknown>\n#15 0x557f3a682e88 <unknown>\n#16 0x557f3a683059 <unknown>\n#17 0x557f3a693843 <unknown>\n#18 0x7f58aac94b43 <unknown>","statusTrace":"self = <test_create_an_account.TestCreateAnAccount object at 0x7f612a81ef50>\ncreate_account_page = <pages.create_an_account_page.CreateAccountPage object at 0x7f612b3c24d0>\n\n    @allure.title('test 04.02.07 create an account with incorrect confirm password')\n    def test_tc_04_02_07_create_account_with_incorrect_confirm_password(self, create_account_page):\n        \"\"\" Verify that customer can't Create An Account with incorrect confirm password\"\"\"\n>       message = create_account_page.create_with_incorrect_confirm_password()\n\ntests/test_create_an_account.py:207: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/create_an_account_page.py:365: in create_with_incorrect_confirm_password\n    self.element_is_visible(self.locators.FIRST_NAME).send_keys('test727')\npages/base_page.py:29: in element_is_visible\n    self.go_to_element(self.element_is_present(locator))\npages/base_page.py:50: in element_is_present\n    return wait(self.driver, timeout).until(EC.presence_of_element_located(locator))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"c0549c4bf3a12709027fe25c496c4ae1\")>\nmethod = <function presence_of_element_located.<locals>._predicate at 0x7f612b496a20>\nmessage = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x557f3a6946b3 <unknown>\nE       #1 0x557f3a36a1e7 <unknown>\nE       #2 0x557f3a3b1566 <unknown>\nE       #3 0x557f3a3b1651 <unknown>\nE       #4 0x557f3a3eea74 <unknown>\nE       #5 0x557f3a3d30ed <unknown>\nE       #6 0x557f3a3ec322 <unknown>\nE       #7 0x557f3a3d2e93 <unknown>\nE       #8 0x557f3a3a5934 <unknown>\nE       #9 0x557f3a3a671e <unknown>\nE       #10 0x557f3a659cb8 <unknown>\nE       #11 0x557f3a65dbf0 <unknown>\nE       #12 0x557f3a66819c <unknown>\nE       #13 0x557f3a65e808 <unknown>\nE       #14 0x557f3a62b27f <unknown>\nE       #15 0x557f3a682e88 <unknown>\nE       #16 0x557f3a683059 <unknown>\nE       #17 0x557f3a693843 <unknown>\nE       #18 0x7f58aac94b43 <unknown>\n\n/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1697332452139,"stop":1697332452233,"duration":94},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"clear_allure_result_folder","time":{"start":1697332452233,"stop":1697332452233,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"driver","time":{"start":1697332618104,"stop":1697332619162,"duration":1058},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"create_account_page","time":{"start":1697332619162,"stop":1697332619316,"duration":154},"status":"passed","steps":[{"name":"Open a browser","time":{"start":1697332619162,"stop":1697332619316,"duration":154},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":1,"attachmentsCount":0,"hasContent":true}],"testStage":{"description":" Verify that customer can't Create An Account with incorrect confirm password","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x557f3a6946b3 <unknown>\n#1 0x557f3a36a1e7 <unknown>\n#2 0x557f3a3b1566 <unknown>\n#3 0x557f3a3b1651 <unknown>\n#4 0x557f3a3eea74 <unknown>\n#5 0x557f3a3d30ed <unknown>\n#6 0x557f3a3ec322 <unknown>\n#7 0x557f3a3d2e93 <unknown>\n#8 0x557f3a3a5934 <unknown>\n#9 0x557f3a3a671e <unknown>\n#10 0x557f3a659cb8 <unknown>\n#11 0x557f3a65dbf0 <unknown>\n#12 0x557f3a66819c <unknown>\n#13 0x557f3a65e808 <unknown>\n#14 0x557f3a62b27f <unknown>\n#15 0x557f3a682e88 <unknown>\n#16 0x557f3a683059 <unknown>\n#17 0x557f3a693843 <unknown>\n#18 0x7f58aac94b43 <unknown>","statusTrace":"self = <test_create_an_account.TestCreateAnAccount object at 0x7f612a81ef50>\ncreate_account_page = <pages.create_an_account_page.CreateAccountPage object at 0x7f612b3c24d0>\n\n    @allure.title('test 04.02.07 create an account with incorrect confirm password')\n    def test_tc_04_02_07_create_account_with_incorrect_confirm_password(self, create_account_page):\n        \"\"\" Verify that customer can't Create An Account with incorrect confirm password\"\"\"\n>       message = create_account_page.create_with_incorrect_confirm_password()\n\ntests/test_create_an_account.py:207: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/create_an_account_page.py:365: in create_with_incorrect_confirm_password\n    self.element_is_visible(self.locators.FIRST_NAME).send_keys('test727')\npages/base_page.py:29: in element_is_visible\n    self.go_to_element(self.element_is_present(locator))\npages/base_page.py:50: in element_is_present\n    return wait(self.driver, timeout).until(EC.presence_of_element_located(locator))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"c0549c4bf3a12709027fe25c496c4ae1\")>\nmethod = <function presence_of_element_located.<locals>._predicate at 0x7f612b496a20>\nmessage = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x557f3a6946b3 <unknown>\nE       #1 0x557f3a36a1e7 <unknown>\nE       #2 0x557f3a3b1566 <unknown>\nE       #3 0x557f3a3b1651 <unknown>\nE       #4 0x557f3a3eea74 <unknown>\nE       #5 0x557f3a3d30ed <unknown>\nE       #6 0x557f3a3ec322 <unknown>\nE       #7 0x557f3a3d2e93 <unknown>\nE       #8 0x557f3a3a5934 <unknown>\nE       #9 0x557f3a3a671e <unknown>\nE       #10 0x557f3a659cb8 <unknown>\nE       #11 0x557f3a65dbf0 <unknown>\nE       #12 0x557f3a66819c <unknown>\nE       #13 0x557f3a65e808 <unknown>\nE       #14 0x557f3a62b27f <unknown>\nE       #15 0x557f3a682e88 <unknown>\nE       #16 0x557f3a683059 <unknown>\nE       #17 0x557f3a693843 <unknown>\nE       #18 0x7f58aac94b43 <unknown>\n\n/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException","steps":[{"name":"Create an account with incorrect confirm password","time":{"start":1697332619317,"stop":1697332624440,"duration":5123},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x557f3a6946b3 <unknown>\n#1 0x557f3a36a1e7 <unknown>\n#2 0x557f3a3b1566 <unknown>\n#3 0x557f3a3b1651 <unknown>\n#4 0x557f3a3eea74 <unknown>\n#5 0x557f3a3d30ed <unknown>\n#6 0x557f3a3ec322 <unknown>\n#7 0x557f3a3d2e93 <unknown>\n#8 0x557f3a3a5934 <unknown>\n#9 0x557f3a3a671e <unknown>\n#10 0x557f3a659cb8 <unknown>\n#11 0x557f3a65dbf0 <unknown>\n#12 0x557f3a66819c <unknown>\n#13 0x557f3a65e808 <unknown>\n#14 0x557f3a62b27f <unknown>\n#15 0x557f3a682e88 <unknown>\n#16 0x557f3a683059 <unknown>\n#17 0x557f3a693843 <unknown>\n#18 0x7f58aac94b43 <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/create_an_account_page.py\", line 365, in create_with_incorrect_confirm_password\n    self.element_is_visible(self.locators.FIRST_NAME).send_keys('test727')\n    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/base_page.py\", line 29, in element_is_visible\n    self.go_to_element(self.element_is_present(locator))\n                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/base_page.py\", line 50, in element_is_present\n    return wait(self.driver, timeout).until(EC.presence_of_element_located(locator))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[{"name":"Find a visible element","time":{"start":1697332619317,"stop":1697332624439,"duration":5122},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x557f3a6946b3 <unknown>\n#1 0x557f3a36a1e7 <unknown>\n#2 0x557f3a3b1566 <unknown>\n#3 0x557f3a3b1651 <unknown>\n#4 0x557f3a3eea74 <unknown>\n#5 0x557f3a3d30ed <unknown>\n#6 0x557f3a3ec322 <unknown>\n#7 0x557f3a3d2e93 <unknown>\n#8 0x557f3a3a5934 <unknown>\n#9 0x557f3a3a671e <unknown>\n#10 0x557f3a659cb8 <unknown>\n#11 0x557f3a65dbf0 <unknown>\n#12 0x557f3a66819c <unknown>\n#13 0x557f3a65e808 <unknown>\n#14 0x557f3a62b27f <unknown>\n#15 0x557f3a682e88 <unknown>\n#16 0x557f3a683059 <unknown>\n#17 0x557f3a693843 <unknown>\n#18 0x7f58aac94b43 <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/base_page.py\", line 29, in element_is_visible\n    self.go_to_element(self.element_is_present(locator))\n                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/base_page.py\", line 50, in element_is_present\n    return wait(self.driver, timeout).until(EC.presence_of_element_located(locator))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[{"name":"Find a present element","time":{"start":1697332619317,"stop":1697332624438,"duration":5121},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x557f3a6946b3 <unknown>\n#1 0x557f3a36a1e7 <unknown>\n#2 0x557f3a3b1566 <unknown>\n#3 0x557f3a3b1651 <unknown>\n#4 0x557f3a3eea74 <unknown>\n#5 0x557f3a3d30ed <unknown>\n#6 0x557f3a3ec322 <unknown>\n#7 0x557f3a3d2e93 <unknown>\n#8 0x557f3a3a5934 <unknown>\n#9 0x557f3a3a671e <unknown>\n#10 0x557f3a659cb8 <unknown>\n#11 0x557f3a65dbf0 <unknown>\n#12 0x557f3a66819c <unknown>\n#13 0x557f3a65e808 <unknown>\n#14 0x557f3a62b27f <unknown>\n#15 0x557f3a682e88 <unknown>\n#16 0x557f3a683059 <unknown>\n#17 0x557f3a693843 <unknown>\n#18 0x7f58aac94b43 <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/base_page.py\", line 50, in element_is_present\n    return wait(self.driver, timeout).until(EC.presence_of_element_located(locator))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[{"name":"locator","value":"('id', 'firstname')"},{"name":"timeout","value":"5"}],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true}],"attachments":[],"parameters":[{"name":"locator","value":"('id', 'firstname')"},{"name":"timeout","value":"10"}],"shouldDisplayMessage":false,"stepsCount":1,"attachmentsCount":0,"hasContent":true}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":2,"attachmentsCount":0,"hasContent":true}],"attachments":[{"uid":"1bc3b910bcc229e2","name":"Screenshot","source":"1bc3b910bcc229e2.png","type":"image/png","size":14924},{"uid":"2395cad2610396c2","name":"HTML source","source":"2395cad2610396c2.html","type":"text/html","size":571}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":3,"attachmentsCount":2,"hasContent":true},"afterStages":[{"name":"driver::0","time":{"start":1697332624615,"stop":1697332624675,"duration":60},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"clear_allure_result_folder::0","time":{"start":1697335302392,"stop":1697335302392,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"labels":[{"name":"epic","value":"Create An Account"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_create_an_account"},{"name":"subSuite","value":"TestCreateAnAccount"},{"name":"host","value":"fv-az453-313"},{"name":"thread","value":"2952-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_create_an_account"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"815570a836cc98b6.json","parameterValues":[]}